test/test_jit_cuda_fuser.py

```diff
@@ -767,43 +767,37 @@ def add(x: torch.Tensor, other: torch.Tensor, alpha: float):
         self._run_helper(add_jit, add, x, y, 2.0)
 
         def clamp0(x: torch.Tensor, f: float):
-            o = torch.rand_like(x)
-            o = o * torch.clamp(x, min=f)
+            o = 1. * torch.clamp(x, min=f)
```

Comment:
Why multiply by `1.`?

Reply:
It triggers a fusion in the forward and backwards pass with an operation that doesn't pollute the operation we are really trying to test. Actually, triggering a backward pass doesn't matter in this instance since we aren't attempting to test a composite op like layer norm and I think _run_helper only triggers the forward pass.  Do you have a preferred operation to generate a fusion?

Reply:
So, just having something like `o = torch.clamp(x. min=f)` doesn't generate a fusion? I don't have any preferred way, I was just curious why it has the multiplication.

Reply:
That is correct, we need two or more fuse-able operations in a row in order for the NVFuser to get exercised.